website/src/components/Showcase/Card.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect } from 'react';
+import styles from '../../pages/showcase.module.css';
+// import cardList from './AppsList';
+import cardList from '../../../showcase.json';
+import Link from '@docusaurus/Link';
+
+function Card({ title, image, link1, link2 }) {
+  return (
+    <section>
```

Comment:
Monstrous markup. You should be fine just with
`li > (div > img) + span + a + a`, 3 levels at most. Or even `li > img + span + a + a` 2 levels.

Reply:
Thank you for re-working this one. Yes, I was using too many divs in this file and I learned a lot from seeing your changes and how simple it is to just use an <li>'s. Appreciate this very much!